src/pages/index.js

```diff
@@ -5,168 +5,225 @@ import Link from '@docusaurus/Link';
+      <div className={styles.spacer}></div>
     </div>
   );
 }
 
 function SelectTutorial() {
   const [clicked, setClicked] = React.useState(false);
   return (
-    <div className="flex flex-col sm:flex-row justify-between mb-8 sm:mb-8 items-center">
+    <div className='flex flex-col sm:flex-row justify-between mb-8 sm:mb-8 items-center'>
       {clicked ? (
-        <div className="inline-flex md:-mt-8">
-          <a className="mr-8" href="/docs/go/run-your-first-app-tutorial">
+        <div className='inline-flex md:-mt-8'>
+          <a className='mr-8' href='/docs/go/run-your-first-app-tutorial'>
             <img
-              aria-label="go SDK"
+              aria-label='go SDK'
               className={styles.sdkTakeMeToTutorialLogo}
-              src={useBaseUrl("img/logo-go.png")}
-              alt="logo"
+              src={useBaseUrl('img/logo-go.png')}
+              alt='logo'
             />
           </a>
-          <a className="" href="/docs/java-run-your-first-app">
+          <a className='' href='/docs/java-run-your-first-app'>
             <img
-              aria-label="java SDK"
+              aria-label='java SDK'
               className={styles.sdkTakeMeToTutorialLogo}
-              src={useBaseUrl("img/logo-java.png")}
-              alt="logo"
+              src={useBaseUrl('img/logo-java.png')}
+              alt='logo'
             />
           </a>
         </div>
       ) : (
         <button
-          className={clsx('button button--outline button--secondary button--lg',
-          styles.getStarted,
+          className={clsx(
+            'button button--outline button--secondary button--lg',
+            styles.getStarted
           )}
-          onClick={() => setClicked(true)} >
+          onClick={() => setClicked(true)}
+        >
           Run my first app! →
         </button>
       )}
     </div>
   );
 }
 
+function TemporalCloud() {
+  return (
+    <div className={styles.formFeature}>
+      <h2>☁️ Temporal Cloud</h2>
+      <p>
+        Temporal Cloud is a fully managed cloud offering from the company behind
+        Temporal's open source. We offer:
+        <ul>
+          <li>
+            <strong>Updates</strong>: Automatic updates with latest releases.
+          </li>
+          <li>
+            <strong>Experience</strong>: Have run this technology in production
+            since our origins at Uber.
+          </li>
+          <li>
+            <strong>Scale</strong>: Our design partners are
+            multi-tens-of-billion valuation publicly listed companies.
+          </li>
+          <li>
+            <strong>Dependencies</strong>: No more managing dependencies like
+            Elasticsearch, Cassandra or MySQL.
+          </li>
+          <li>
+            <strong>Prioritization</strong>: Dedicated channels for support and
+            product feedback.
+          </li>
+          <li>
+            <strong>Future improvements</strong>: to be discussed as needed
```

Comment:
I think this one can be removed.
Doesn't really say much beyond "automatic updates" which is covered above.